lib/view_model/dashboard/dashboard_view_model.dart

```diff
@@ -301,13 +301,8 @@ abstract class DashboardViewModelBase with Store {
   bool get isTestnet => wallet.type == WalletType.bitcoin && bitcoin!.isTestnet(wallet);
 
   @computed
-  bool get hasRescan =>
-      wallet.type == WalletType.bitcoin ||
-      wallet.type == WalletType.monero ||
-      wallet.type == WalletType.wownero ||
-      wallet.type == WalletType.decred ||
-      wallet.type == WalletType.haven;
-
+  bool get hasRescan => wallet.hasRescan;
+     
```

Comment:
nit: There are some "bad white spaces" here on line 305.

Reply:
Resolved, I had to rebase and edit.